framework/source/class/qx/ui/basic/Image.js

```diff
@@ -508,6 +508,10 @@ qx.Class.define("qx.ui.basic.Image",
       // Detect if the image registry knows this image
       if (ResourceManager.isFontUri(source)) {
         this.__setManagedImage(contentEl, source);
+        var color = this.getTextColor();
+        if (qx.lang.Type.isString(color)) {
+          this._applyTextColor(color, null);
+        }
```

Comment:
Is that the only relevant property that hasn't yet been set? It seems that if text color hasn't been set, it's likely that other properties too have not been set...

Reply:
The problem is only with textColor as the _applyTextColor method checks if we are working with a fontIcon ... but that test fails at creation time as the source gets set later than the textColor when theming ... or rather randomly ... I think we had that problem before ... that properties depend on each other and thus the orther they get applied matters ... this patch works around this by just re-applying the text color once the styling call comes